src/components/plots/gridPlot.ts

```diff
@@ -69,17 +66,14 @@ export module Plots {
       return this;
     }
 
-    public y(): Plots.AccessorScaleBinding<any, number>;
-    public y(y: number | _Accessor): Grid;
-    public y(y: any | _Accessor, scale: Scale<any, number>): Grid;
-    public y(y?: number | _Accessor | any, scale?: Scale<any, number>): any {
+    public y(y?: number | Accessor<number> | any, scale?: Scale<any, number>): any {
```

Comment:
Same comments regarding using a generic here, by having the `Grid` class also accept generics.